### ISSUE TITLE
Do translations need to have full HTML markup/logic?

### ISSUE THREAD

**Issue by sampaiodiego** [RocketChat/rocketchat.github.io#138]
Current translations are a full copy from english page files, just translating texts.

But with the increasing number of translated languages, it will be a nightmare to edit anything on any page, because you'll have to spread the change across on translated languages.

The ideal solution on my opinion would be having just one source of HTML markup and logic with place holders for translated texts. Localization files would then have only the strings that need to be translated.

**Comment by SeanPackham** [comment]
Yes this would be the ideal setup. Let's chat more about it at the summit.

**Comment by sampaiodiego** [comment]
should we keep this open and then close it when we have a final decision or implementation?

**Comment by engelgabriel** [comment]
I agree with @sampaiodiego let's keep this open until we deploy the ideal setup.

**Comment by SeanPackham** [comment]
I have discussed this with Significa and they are implementing a improved translation system.
